naviguide.cc

```diff
@@ -326,16 +379,19 @@ rd_init(const QString& fname)
 
   ng_read_file_header();
 
-  if (NULL != process) {
-    if (!case_ignore_strcmp(process, "wp")) {
-      process_rte = 0;
-    }
-    if (!case_ignore_strcmp(process, "rte")) {
-      process_rte = 1;
-    }
+  if (!case_ignore_strcmp(read_as, "wp")) {
```

Comment:
These could be QString::compare(s1, s2, Qt::CaseInsensitive) and chained elses, but it's probably better to just add everything all the time independent of options.